modules/core/server/services/email.server.service.js

```diff
@@ -226,7 +226,7 @@ exports.sendSupportRequest = function (replyTo, supportRequest, callback) {
     name: 'Trustroots Support', // `To:`
     email: config.supportEmail, // `To:`
     replyTo: replyTo,
-    subject: 'Support request',
+    subject: 'Support request from ' + supportRequest.username + ' (' + supportRequest.displayName + ')',
```

Comment:
Note that `username` or `displayName` are not necessarily available (it's possible to submit support form while not signed in), so these need to be conditional.